lib/matplotlib/widgets.py

```diff
@@ -1934,8 +1934,15 @@ def onmove(self, event):
             self.linev.set_visible(False)
             self.lineh.set_visible(False)
             if self.needclear:
+                if self.useblit and self.background is not None:
```

Comment:
If both `needclear` and `useblit` are `True`, is it possible for `background` to be `None`?  Looking at the code I do not think so, but maybe I missed something.